### ISSUE TITLE
Untuntu/Debian package error

### ISSUE THREAD

**Issue by alemrsrl** [wkhtmltopdf/packaging#94]
when i'm try  dpkg -i  wkhtmltox_0.12.6-1.buster_ppc64le.deb i receive the error: package architecture (ppc64le) does not match system (ppc64el)

i want to build wkhtmltopdf form my ubuntu installend on IBM Power Server (Power 9 processor), but i don't find any solution for build on my system

**Comment by ashkulz** [comment]
You're right, there was a bug in the architecture name which was fixed in 01d80d7c32e263655fe48b842804058058f984a6 -- I just haven't rebuilt the packages yet, can you build locally and try?

**Comment by alemrsrl** [comment]
Sorry but i can't build it because I don't have the necessary privileges on my system to build it.
could you do it for me (Debian and Ubuntu)?

**Comment by ashkulz** [comment]
You can build it on a regular Intel machine and copy it over, see the instructions in README.

**Comment by ashkulz** [comment]
Builds have been published to the website: see wkhtmltopdf/wkhtmltopdf@e59bfcce3e8db83341264bdd75d996319402ccff.
